failed_interview.py

```diff
@@ -0,0 +1,193 @@
+import gspread
+import operator
+from oauth2client.service_account import ServiceAccountCredentials
+import time
+import constant
+
+scopes = ["https://spreadsheets.google.com/feeds", 'https://www.googleapis.com/auth/spreadsheets',
+          "https://www.googleapis.com/auth/drive.file", "https://www.googleapis.com/auth/drive"]
+
+months = {'Jan': 3, 'Feb': 4, 'Mar': 5, 'Apr': 6, 'May': 7, 'Jun': 8, 'July': 9, 'Aug': 10, 'Sep': 11, 'Oct': 12,
+          'Nov': 13, 'Dec': 14}
+cred = ServiceAccountCredentials.from_json_keyfile_name("cred.json", scopes)
+client = gspread.authorize(cred)
+dashboard = client.open(constant.SHEET_NAME).get_worksheet_by_id(constant.DASHBOARD_SHEET_ID)
+client_interview = client.open(constant.SHEET_NAME).get_worksheet_by_id(constant.CLIENT_INTERVIEW)
+
```

Comment:
put these lines inside any funtion , because if you import this file , these lines will get executed automatically